packages/stems/src/assets/styles/animations.css

```diff
@@ -8,5 +8,6 @@
 :root,
 root {
   --quick: 0.07s ease-in-out;
-  --calm: 0.2s ease-in-out;
+  --expressive: 0.2s ease-in-out;
```

Comment:
does anything else use the calm var? change makes sense, but want to make sure that nothing else is affected

Reply:
yes i went ahead and updated all usages :)